src/runtime/process-worker.ts

```diff
@@ -149,7 +150,9 @@ export default class ProcessWorker implements TinypoolWorker {
 
     // The forked child_process adds event listener on `process.on('message)`.
     // This requires manual unreffing of its channel.
-    this.process.channel?.unref()
+    if (this.process.channel && hasUnref(this.process.channel)) {
+      this.process.channel?.unref()
+    }
```

Comment:
This one's quite unfortunate. What does Bun put in `unref`? Why is it not function?

Let's add some one-liner comment about it.

Reply:
Could we simplify this change to:

https://github.com/tinylibs/tinypool/blob/5db47eeae31dfd39a9f9baed185d4d1cf78460e6/src/runtime/process-worker.ts#L152

```diff
- this.process.channel?.unref()
+ this.process.channel?.unref?.()
```